packages/paypal-js/types/v6/components/base-component.d.ts

```diff
@@ -15,7 +15,19 @@ export type OnCompleteData = {
     paymentSessionState: "approved" | "canceled" | "error";
 };
 
-export type OnErrorData = Error;
+export type PayPalError = Error & {
```

Comment:
Happy to update these to use `interface` if that makes more sense.

Reply:
@kand @EvanReinstein I switched to `interface` for PayPalError and PayPalWarning